force-app/main/default/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -34,6 +34,9 @@ import PAYMENT_LAST_4 from '@salesforce/schema/DataImport__c.Payment_Card_Last_4
 import DATA_IMPORT_ID from '@salesforce/schema/DataImport__c.Id';
 import DATA_IMPORT from '@salesforce/schema/DataImport__c';
 
+import commonPaymentServices from '@salesforce/label/c.commonPaymentServices';
```

Comment:
I think these labels should be included in geLabelService rather than imported separately